src/modules/BootArch.rb

```diff
@@ -36,13 +36,14 @@ def main
     end
 
     # list of regexp to match kernel params that should be added
-    # from installation to running kernel on s390 (bsc#1086665)
+    # from installation to running kernel on s390 (bsc#1086665, bsc#1210525)
     S390_WHITELIST = [
       /net\.ifnames=\S*/,
       /fips=\S*/,
       /mitigations=\S*/,
       /rd\.zdev=\S*/,
-      /zfcp\.allow_lun_scan=\S*/
+      /zfcp\.allow_lun_scan=\S*/,
+      /cio_ignore=\S*/
```

Comment:
I would add here commend why we keep cio_ignore ( so point to bug there and also mention that we prefer one from installation and otherwise we propose it ).